### PR TITLE
Puts naga footstep on the organ, allows for (scuffed) unapplying

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

Title.

Closes https://github.com/Bubberstation/Bubberstation/issues/1746

Previously, it was done on job. Which is. Weird, but taur organs weren't subtyped properly, so we can fix it now.
GetElement doesnt exist, so I cant store the existing footstep for reapplying later, so we jsut guess with human footsteps.

Also, changes a weird edit title. Boob edit? Really?

## Why It's Good For The Game

bugs bad!!!!!

## Proof Of Testing

https://github.com/user-attachments/assets/bb39ff2e-3224-46b0-aec7-14f283309dde

## Changelog

:cl:
fix: Nagas off-station or restored with the SAD should now properly have their footstep sound
fix: Losing your naga lower body now tries to restore normal footsteps
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
Drafted pending testing

File: modular_zubbers/code/modules/taur_mechanics/serpentine_taur.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/obj/item/organ/external/taur_body/serpentine/on_mob_insert(mob/living/carbon/organ_owner, special, movement_flags)
+	. = ..()
+
+	RemoveElement(/datum/element/footstep, FOOTSTEP_MOB_HUMAN, 0.6, -6)
```
Comment (nikothedude) [inline modular_zubbers/code/modules/taur_mechanics/serpentine_taur.dm:4]: tbh i dont know what these args are for and i dont care

Comment (Majkl-J) [reply modular_zubbers/code/modules/taur_mechanics/serpentine_taur.dm:4]: Remove element just takes the default argument so this should be `RemoveElement(/datum/element/footstep)`

What makes me happy is that I didn't approve whatever this code was originally

File: modular_zubbers/code/modules/taur_mechanics/serpentine_taur.dm [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	AddElement(/datum/element/footstep, FOOTSTEP_MOB_SNAKE, 15, -6)
+
+/obj/item/organ/external/taur_body/serpentine/on_mob_remove(mob/living/carbon/organ_owner, special, movement_flags)
+	. = ..()
+
+	RemoveElement(/datum/element/footstep, FOOTSTEP_MOB_SNAKE, 15, -6)
+	AddElement(/datum/element/footstep, FOOTSTEP_MOB_HUMAN, 0.6, -6)
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/taur_mechanics/serpentine_taur.dm:11]:
```suggestion
	RemoveElement(/datum/element/footstep)
	AddElement(/datum/element/footstep, FOOTSTEP_MOB_HUMAN, 0.6, -6)
```

File: modular_zubbers/code/modules/taur_mechanics/serpentine_taur.dm (hunk 1 above)
Comment (Majkl-J) [inline modular_zubbers/code/modules/taur_mechanics/serpentine_taur.dm:4]:
```suggestion
	RemoveElement(/datum/element/footstep)
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Comment by nikothedude** [comment]
ok shes all gassed up and ready to go

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**